### PR TITLE
Update README file taking out extra information. I added project depe…

### PR DESCRIPTION
**Pull request by nestornav**
A brief update to get a more clearly README. All the extra information was moved to the wiki.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +79,7 @@ jQuery(document.getElementById('#some-panel'));
 
 A meta-function can avoid re-writing the caller expression by returning null instead of the AST.
 
+You could go to PumaScript wiki and find more examples.
```
Comment (emravera) [inline README.md]: We should add a Link here.

Comment (nestornav) [reply README.md]: Okay I'll add it. Thanks.

**Comment by Agu-GC** [comment]
Hi Nestor! We could update the URL used to clone the repository.
(https://github.com/pumascript/puma.git)

**Comment by nestornav** [comment]
Nice catch @Agu-GC . Thanks

**Pull request merged**